staging/src/k8s.io/kubectl/pkg/cmd/create/create_clusterrolebinding.go

```diff
@@ -100,10 +100,11 @@ func NewCmdCreateClusterRoleBinding(f cmdutil.Factory, ioStreams genericclioptio
 	cmdutil.AddDryRunFlag(cmd)
 	cmd.Flags().StringVar(&o.ClusterRole, "clusterrole", "", i18n.T("ClusterRole this ClusterRoleBinding should reference"))
 	cmd.MarkFlagRequired("clusterrole")
-	cmd.Flags().StringArrayVar(&o.Users, "user", o.Users, "Usernames to bind to the clusterrole")
+	cmd.LocalFlags().StringArrayVar(&o.Users, "user", o.Users, "Usernames to bind to the clusterrole")
```

Comment:
hmmm.... can you verify how many flags we have shadowed? I'd consider renaming those arguments, rather than fixing it this way.

Reply:
ok. What about rename user to users ?

Reply:
Have been renamed to `users`, if you have better name. Welcome to comment~😳